docs/src/main/tut/idl-generation.md

```diff
@@ -221,6 +221,8 @@ Just like `idlGen`, `srcGen` and `srcGenFromJars` has some configurable settings
   * All the source folders specified in the `srcGenSourceDirs` setting.
 * **`srcGenTargetDir`**: the Scala target base directory, where the `srcGen` task will write the Scala files in subdirectories/packages based on the namespaces of the IDL files. By default: `Compile / sourceManaged`, typically `target/scala-2.12/src_managed/main/`.
 * **`genOptions`**: additional options to add to the generated `@service` annotations, after the IDL type. Currently only supports `"Gzip"`.
+* **`idlGenCodecImports`**: additional imports to add on top to the generated service files. This property can be used for importing extra encoders/decoders for your services. By default: `List(BigDecimalAvroMarshallers, JavaTimeDateAvroMarshallers)`. The `JodaDateTimeAvroMarshallers` is also available, but it needs the dependency `frees-rpc-marshallers-jodatime`. You can also specify custom imports with the following:
```

Comment:
Maybe we can rename this to `idlGenMarshallerImports`, to be fully aligned, thoughts?

Reply:
Agree